### PR TITLE
up to date French translation

### PR DESCRIPTION
**Pull request by maieul**
Hi, I hope that is ok.

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -3,51 +3,48 @@
...
 #: src/backup_base.vala:99
 msgid "Ready"
-msgstr ""
+msgstr "Prêt"
 
 #. TRANSLATORS This is the name for the backend that allows to store the backups in a folder, instead of choosing a disk.
 #: src/backup_folder.vala:68
 msgid "Store backups in a folder"
-msgstr ""
+msgstr "Sauvegarder dans un dossier"
```
Comment (funilrys) [inline po/fr.po]: "Sauvegarder les sauvegardes dans un dossier" is better.

File: po/fr.po [hunk 2]
```diff
@@ -57,86 +54,75 @@ msgstr "Suppression d’anciennes sauvegardes"
 #: src/backup_folder.vala:357 src/backup_rsync.vala:370
 #, c-format
 msgid "Backup done. Elapsed time: %d:%02d"
-msgstr ""
+msgstr "Sauvegarde effectuée. Temps restant : %d:%02d"
```
Comment (funilrys) [inline po/fr.po]: Elapsed == `écoulé`.

File: po/fr.po [hunk 3]
```diff
@@ -57,86 +54,75 @@ msgstr "Suppression d’anciennes sauvegardes"
...
+msgstr "Impossible de restaurer le fichier %s"
 
 #: src/backup_folder.vala:422 src/backup_rsync.vala:435
-#, fuzzy, c-format
+#, c-format
 msgid "Failed to delete aborted backups: %s"
-msgstr ""
-"Impossible de libérer de l’espace disque pour démarrer une sauvegarde. "
-"Interruption de la sauvegarde.\n"
+msgstr "Impossible de supprimer la sauvegarde échouée : %s."
```
Comment (funilrys) [inline po/fr.po]: En pluriel c'est mieux `backups` l'ai.

File: po/fr.po [hunk 4]
```diff
@@ -57,86 +54,75 @@ msgstr "Suppression d’anciennes sauvegardes"
...
 #: src/backup_rsync.vala:557 src/backup_rsync.vala:558
-#, fuzzy
 msgid "Syncing disk"
-msgstr "Synchronisation du disque\n"
+msgstr "Synchronisation du disque"
 
 #: src/backup_folder.vala:551 src/backup_rsync.vala:564
 msgid "Failed to launch sync command"
-msgstr ""
+msgstr "Échec du chargement de la commande sync"
```
Comment (funilrys) [inline po/fr.po]: `de l'éxecution` may be better? :thinking:

File: po/fr.po [hunk 5]
```diff
@@ -57,86 +54,75 @@ msgstr "Suppression d’anciennes sauvegardes"
...
 msgstr ""
-"Impossible de libérer de l’espace disque pour démarrer une sauvegarde. "
-"Interruption de la sauvegarde.\n"
+"Impossible de renommer le dossier de sauvegarde. Interruption de la "
+"sauvegarde."
 
 #: src/backup_folder.vala:587 src/backup_rsync.vala:600
 msgid "Failed to launch final sync command"
-msgstr ""
+msgstr "Échec du chargement de la commande sync"
```
Comment (funilrys) [inline po/fr.po]: This one is not the proper translation ... Even literally ...

File: po/fr.po [hunk 6]
```diff
@@ -642,6 +622,10 @@ msgid ""
 "enable it, set the <i>gconf</i> key 'enable-folder-backend' to <i>true</i>. "
 "If you don't know how to do this, probably you should not use this backend."
 msgstr ""
+"Le <i>dossier de sauvegarde</i> n'est considéré comme fonctionnel que pour "
+"les utilisateurs avancés. Pour l'activer, configurer la clef <i>gconf</i> "
+"'enable-folder-backend' à <i>true</i>. Si vous ne savez pas comment faire "
+"cela, vous ne devriez probablement utiliser ce mode de sauvegarde."
```
Comment (funilrys) [inline po/fr.po]: `probablement pas`

File: po/fr.po [hunk 7]
```diff
@@ -657,10 +641,23 @@ msgid ""
 "disk. Or when using the same physical disk, that (s)he understands that "
 "there is no true backup of the data."
 msgstr ""
+"Le système <i>conserver les sauvegarde dans un dossier</i> peut être "
```
Comment (funilrys) [inline po/fr.po]: `sauvegardes`

File: po/fr.po [hunk 8]
```diff
@@ -657,10 +641,23 @@ msgid ""
...
+"considéré comme un choix assez dangereus, parce qu'il permet de faire ce qui "
```
Comment (funilrys) [inline po/fr.po]: `dangereux`

File: po/fr.po [hunk 9]
```diff
@@ -657,10 +641,23 @@ msgid ""
...
+"est considéré comme une « mauvaise pratique » pour une sauvegarde sécurisée. "
+"En particulier, il permet de stocker les sauvegarde sur le même disque "
```
Comment (funilrys) [inline po/fr.po]: `sauvegardes`

File: po/fr.po (hunk 2 above)
Comment (maieul) [reply po/fr.po]: effectivement.

File: po/fr.po (hunk 1 above)
Comment (maieul) [reply po/fr.po]: "Sauvegarder les sauvegardes" is pleonastic. Mabyer "conserver les sauvegardes".

File: po/fr.po (hunk 4 above)
Comment (maieul) [reply po/fr.po]: du lancement

File: po/fr.po (hunk 5 above)
Comment (maieul) [reply po/fr.po]: I din't understand in english what does it mean.

File: po/fr.po (hunk 6 above)
Comment (maieul) [reply po/fr.po]: oui

File: po/fr.po (hunk 7 above)
Comment (maieul) [reply po/fr.po]: oui

File: po/fr.po (hunk 8 above)
Comment (maieul) [reply po/fr.po]: oui

File: po/fr.po (hunk 9 above)
Comment (maieul) [reply po/fr.po]: oui

**Comment by maieul** [comment]
thanks, I commented your comments. I will wait for answer before modifiying my local file.

**Comment by maieul** [comment]
hum,. as explained by @funilrys there was some typo / problem in translation. I will reopen a new merge request tonight or tomorrow

**Comment by rastersoft** [comment]
Ok. I will wait before building the new packages.

**Pull request merged**